### ISSUE TITLE
Questions about Performance Reproducing

### ISSUE THREAD

**Issue by NAM-hj** [Jiaxin-Ye/TIM-Net_SER#13]
Thank you for sharing your great work! It is really helpful.
I'd like to ask you for Performance reproducing on the EmoDB in the paper (~95%). But, I can only get around 85% average test accuracy. (I ran main.py several times without changing any of it.) Would you kindly let me know how you can achieve that result?

Thank you

**Comment by Jiaxin-Ye** [comment]
Dear @NAM-hj ,

Thank you for your interest in this project and your valuable suggestions. I sincerely apologize for any inconvenience caused to you. You might consider **extending the training epochs** to achieve better performance, as small epoch numbers may hinder the convergence toward optimal results.

Moreover, we have supplemented more experimental results in the updated README file providing further details on the experiment specifics. Feel free to refer to it for more information. I greatly appreciate your feedback, which helps us continually refine and enhance the quality of our work. If you have any further questions or feedback, please don't hesitate to reach out to me. Thank you very much for your understanding and support.

Best regards,

Jiaxin Ye

**Comment by hecongqing** [comment]
btw,   I repeat  the experiment , the  feature is downloaded from the file,  and the training code is 
```
python main.py --mode train --data EMODB  --split_fold 10 --random_seed 666 --epoch 500 --gpu 0
```

I got the performance:
Average UAR: 0.8946531642960215
Average WAR: 0.8970649895178198
